### ISSUE TITLE
Consider including CSS for Dark Mode

### ISSUE THREAD

**Issue by petecooper** [textpattern/textpattern-default-theme#328]
### Is your feature request related to a problem?

No.

### What is the feature?

Dark Mode CSS. See:

* https://paulmillr.com/posts/using-dark-mode-in-css/

**Comment by philwareham** [comment]
Ok thanks for the tip - looks easy to set up so I’ll give it a try soon.

**Comment by philwareham** [comment]
I've played with this feature a bit in Safari Tech Preview, but whilst interesting I don't see it as appropriate for a standard website.

To explain, I run my Mac in Dark Mode here but I don't want or expect sites I visit to base their colour design on my UI preference (I also changed the hideous dark display of emails back to standard white background in my Mac Mail app for that very reason).

However, where I do find it useful (and really what it is intended for) is for web apps - such as Textpattern admin theme. So I'll close here and investigate further in that theme instead.

Cheers for the heads-up though.
